test/index.js

```diff
@@ -56,90 +56,95 @@ describe('hapi-setup Plugin', function () {
 
         var connections = res.result.connections;
 
-        expect(connections[0].routes).to.deep.equal([
-          {
-            method: 'GET',
-            path: '/about',
-            plugin: null
-          },
-          {
-            method: 'GET',
-            path: '/foo-no-labels',
-            plugin: 'foo'
-          },
-          {
-            method: 'GET',
-            path: '/foo-private-label',
-            plugin: 'foo'
-          },
-          {
-            method: 'GET',
-            path: '/foo-public-label',
-            plugin: 'foo'
-          },
-          {
-            method: 'GET',
-            path: '/server-no-labels',
-            plugin: null
-          },
-          {
-            method: 'GET',
-            path: '/server-private-label',
-            plugin: null
-          },
-          {
-            method: 'GET',
-            path: '/server-public-label',
-            plugin: null
-          }
-        ]);
+        expect(connections[0].routes).to.deep.equal([{
```

Comment:
I would really like to change this so the order doesn't matter. I had to change this around like 5 or 6 times just because the order changed.